.gitignore

```diff
@@ -7,7 +7,7 @@ covasim/cova_seattle/capacity_analysis
 covasim/cova_seattle/school_analysis
 covasim/cova_seattle/seattle_analysis
 results/
-
+data/
```

Comment:
I was wondering if we wanted to seperate the population data from the virus data?
https://github.com/InstituteforDiseaseModeling/covasim/blob/e1707f931011ca3c6bf72598f3c371ec53036259/.gitignore#L142-L144